src/modrover/strategies/base.py

```diff
@@ -62,22 +62,18 @@ def _as_learner_id(self, cov_ids: tuple[int, ...]) -> LearnerID:
         if not all(map(lambda x: 0 <= x, cov_ids)):
             raise ValueError("Cannot have negative covariate IDs")
 
-        if 0 not in cov_ids:
-            # Intercept always a fixed covariate, present in all models
```

Comment:
If the learner ID exists solely to identify unique combinations of explore_cols, this 0 insert serves no purpose and makes it easier to index out of bounds in rover, hence the removal